packages/cli/src/ui/themes/color-utils.ts

```
@@ -250,3 +250,110 @@ export function interpolateColor(
+    };
+
+    const dataHandler = (data: Buffer) => {
+      // Accumulate response across multiple data events (handles split chunks)
+      response += data.toString();
+
+      // OSC 11 response formats:
+      // ESC ] 11 ; rgb:RRRR/GGGG/BBBB ESC \ (ST terminator - standard)
+      // ESC ] 11 ; rgb:RRRR/GGGG/BBBB BEL   (BEL terminator - legacy terminals)
+      // Match either terminator (case-insensitive hex)
+      const matchST = response.match(
+        /\x1b\]11;rgb:([0-9a-f]{4})\/([0-9a-f]{4})\/([0-9a-f]{4})\x1b\\/i,
```

Biome (2.4.4)

[error] Unexpected control character in a regular expression.

(lint/suspicious/noControlCharactersInRegex)